### ISSUE TITLE
jQuery.fn.find(selector) isn't monitored for noElementsFound

### ISSUE THREAD

**Issue by dylang** [padolsey-archive/jquery.lint--old#13]
Current behavior:
$('.blahblah') triggers noElementsFound.

$(document).find('.blahblah') does not trigger noElementsFound.

Expected behavior:
Both should trigger noElementsFound.  

(It doesn't seem to monitor for bad selectors either.)

**Comment by padolsey** [comment]
This has been fixed. jQlint now monitors the following methods for the noElementsFound check:

```
['find', 'children', 'parent', 'parents',
     'next', 'nextAll', 'prev', 'prevAll',
     'first', 'last', 'closest', 'siblings']
```
